<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menuWindow</class>
 <widget class="QMainWindow" name="menuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogOut">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>340</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(37, 99, 175);
border-radius:8px</string>
    </property>
    <property name="text">
     <string>Cerrar sesi√≥n</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEntidades">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(170, 255, 255);
selection-color: rgb(170, 170, 255);
border-radius:8px;
selection-background-color: rgb(0, 255, 255);
</string>
    </property>
    <property name="text">
     <string>Entidades</string>
    </property>
   </widget>
  </widget>
  <action name="actionEmpleados">
   <property name="text">
    <string>Empleados</string>
   </property>
  </action>
  <action name="actionTesoreria">
   <property name="text">
    <string>Sucursales</string>
   </property>
  </action>
  <action name="actionEntidadessEmpleado">
   <property name="text">
    <string>Empleados</string>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionEntidadeSucursales">
   <property name="text">
    <string>Sucursales</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
